### ISSUE TITLE
trickleComposable not exported

### ISSUE THREAD

**Issue by mwargan** [marcoschulte/vue3-progress#2]
In the docs it says that trickleComposable is provided as a composable, but I can't seem to import it as it doesn't look to exist in the exports https://github.com/marcoschulte/vue3-progress/blob/master/src/index.ts.

**Comment by marcoschulte** [comment]
Hi mwargan,
you are right, never noticed it. I released a new version 0.0.7 which exports trickleComposable. Let me know if you still experience any difficulties.

**Comment by mwargan** [comment]
@marcoschulte thanks! The only reason I wanted to extend it is because I wanted to use a native semantic HTML element for progress. I'm sharing it here as I think it might be a good idea to have this in the package by default

```js
<template>
    <progress v-if="state.active"
        :value="value"
        max="100"></progress>
</template>

<script setup lang="ts">
import { useProgress, trickleComposable } from '@marcoschulte/vue3-progress';

const state = useProgress().state();
const { value } = trickleComposable(state);
</script>

<style scoped>
progress {
    position: absolute;
    top: 0;
    left: 0;
    right: 0;
    z-index: 1000;
}
</style>
```
